plugins/catalog/src/alpha.tsx

```diff
@@ -97,27 +97,30 @@ export function createCatalogFilterExtension<
   configSchema?: PortableSchema<TConfig>;
   loader: (options: { config: TConfig }) => Promise<JSX.Element>;
 }) {
+  const id = `catalog.filter.${options.id}`;
+  const attachTo = { id: 'plugin.catalog.page.index', input: 'filters' };
```

Comment:
How come you lifted this line up here?

Reply:
I think I did it on the search create result item extension following the config schema pattern, but I reverted it in all places 🙂